tests/rhui3_tests/test_atomic_client.py

```diff
@@ -43,100 +51,116 @@ def setup_class():
 
-    def test_07_generate_atomic_ent_cert(self):
+    def test_07_generate_atomic_cert(self):
         '''
            generate an entitlement certificate for the Atomic repo
         '''
-        RHUIManagerClient.generate_ent_cert(connection, [self.atomic_repo_name], "test_atomic_ent_cli", "/root/")
-        Expect.expect_retval(connection, "test -f /root/test_atomic_ent_cli.crt")
-        Expect.expect_retval(connection, "test -f /root/test_atomic_ent_cli.key")
+        RHUIManagerClient.generate_ent_cert(CONNECTION,
+                                            [self.atomic_repo_name],
+                                            "test_atomic_ent_cli",
+                                            "/root/")
+        Expect.expect_retval(CONNECTION, "test -f /root/test_atomic_ent_cli.crt")
+        Expect.expect_retval(CONNECTION, "test -f /root/test_atomic_ent_cli.key")
 
     @staticmethod
     def test_08_create_atomic_pkg():
         '''
            create an Atomic client configuration package
         '''
-        RHUIManager.initial_run(connection)
-        RHUIManagerClient.create_atomic_conf_pkg(connection, "/root", "test_atomic_pkg", "/root/test_atomic_ent_cli.crt", "/root/test_atomic_ent_cli.key")
-        Expect.expect_retval(connection, "test -f /root/test_atomic_pkg.tar.gz")
+        RHUIManager.initial_run(CONNECTION)
+        RHUIManagerClient.create_atomic_conf_pkg(CONNECTION,
+                                                 "/root",
+                                                 "test_atomic_pkg",
+                                                 "/root/test_atomic_ent_cli.crt",
+                                                 "/root/test_atomic_ent_cli.key")
+        Expect.expect_retval(CONNECTION, "test -f /root/test_atomic_pkg.tar.gz")
 
     #def test_09_check_sync_status_of_atomic_repo(self):
     #    '''
     #       check if Atomic repo was synced to pull the content
     #    '''
-    #    RHUIManager.initial_run(connection)
-    #    atomic_repo_version = RHUIManagerRepo.get_repo_version(connection, self.atomic_repo_name)
-    #    RHUIManagerSync.wait_till_repo_synced(connection, Util.format_repo(self.atomic_repo_name, atomic_repo_version))
+    #    RHUIManager.initial_run(CONNECTION)
+    #    atomic_repo_version = RHUIManagerRepo.get_repo_version(CONNECTION, self.atomic_repo_name)
+    #    RHUIManagerSync.wait_till_repo_synced(CONNECTION,
+    #                                          Util.format_repo(self.atomic_repo_name,
+    #                                                           atomic_repo_version))
 
     @staticmethod
     def test_10_install_atomic_pkg():
         '''
            install atomic pkg on atomic host
         '''
-        Util.install_pkg_from_rhua(connection, atomic_cli, "/root/test_atomic_pkg.tar.gz")
+        Util.install_pkg_from_rhua(CONNECTION, ATOMIC_CLI, "/root/test_atomic_pkg.tar.gz")
 
     #@staticmethod
```

Comment:
They were commented out before this commit, too. The explanation is now at the top of the file:

# some tests are commented out because the Atomic Trees repo won't sync reliably
# see RHBZ#1427190

I've now commented out the CDS and HAProxy handling as these nodes aren't currently used further in the test and their setup and teardown only take time. When the situation with the Atomic trees repo gets better, we'll uncomment all the remaining tests.

Reply:
ack